e2e/wdio/wdio-multiremote.conf.ts

```diff
@@ -0,0 +1,49 @@
+import url from 'node:url'
+import path from 'node:path'
+import { config as baseConfig } from './wdio.conf.js'
+
+const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
+
+export const config = {
+    ...baseConfig,
+
+    /**
+     * Sauce specific config
+     */
+    specs: [path.resolve(__dirname, 'headless', 'multiremoteTest.e2e.ts')],
+    exclude: [],
+    capabilities: [
+        {
+            browserA: {
+                capabilities: {
+                    browserName: 'chrome',
+                    browserVersion: 'stable',
+                    'wdio:devtoolsOptions': { headless: true, dumpio: true }
```

Comment:
These will run with WebDriver so those capabilities won't work anymore, let's do:
```suggestion
                    'goog:chromeOptions': { args: ['headless', 'disable-gpu'] }
```